appconfiguration/client/README.md

```diff
@@ -185,14 +185,12 @@ If you would like to become an active contributor to this project please follow
 5. Create new Pull Request
 
 <!-- LINKS -->
-[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/applicationconfig/client/src
-[package]:not-valid-link
-[api_documentation]: not-valid-link
-[azconfig_docs]: https://docs.microsoft.com/en-us/azure/azure-app-configuration/
-[jdk]: https://docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable
-[maven]: https://maven.apache.org/
-[azure_subscription]: https://azure.microsoft.com/en-us/free/
+[api_documentation]: https://aka.ms/java-docs
```

Comment:
Not sure how "Client Libraries" and "Data Libraries" are differentiated. Can we use link http://azure.github.io/azure-sdk-for-java/track2reports/index.html here for easier discovery of the relevant doc for the App Configuration?